practice-app/server/src/tests/data.js

```diff
@@ -0,0 +1 @@
+const userId = new mongoose.Types.ObjectId().toString();
```

Comment:
Do you use this, I could not see a reference.

Reply:
removed file.